lab-ron-dunphy/index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
```

Comment:
Nice job! 

Your commits are a little on the lacking side, so be sure to focus on your git workflow (ACP!)